process/rating/ratingsData.go

```diff
@@ -497,6 +498,10 @@ func (rd *RatingsData) updateRatingsMetrics(epoch uint32) {
 }
 
 func getRatingStepsForEpoch(epoch uint32, ratingStepsPerEpoch []config.RatingSteps) (config.RatingSteps, bool) {
+	slices.SortFunc(ratingStepsPerEpoch, func(a, b config.RatingSteps) bool {
```

Comment:
Sorts inline (alters input array) I think, but should be fine (I assume no concurrency).